main.css

```diff
@@ -0,0 +1,500 @@
+  margin-top: 50px;
+  font-family: 'Monda', sans-serif;
+  margin-bottom: 20px;
+  border-radius: 20px;
+  background-color:#EC9A29;
+  height: 85%;
+}
+#post-list{
+  color: white;
+  font-family: 'Monda', sans-serif;
+  background-color:#EC9A29;
+  border-radius: 20px;
+  height: 80%;
+}
+#post-list > li:hover,
+li:focus-within {
+  background:#A8201A;
+  cursor: pointer;
+}
+
+/* Separate Restaurant Page Below---------------------------- */
+.restpage{
+  display:grid;
+  width: 100%;
+  height: 100%;
+  grid-gap: 1rem;
+  grid-template-columns: 5% 30% 30% 25% 10%;
+  grid-template-rows: 15% 50% 20% 15%;
+  grid-template-areas: 
+  ". header header header navigation "
+  ". main main sidebarList  ."
+  ". commentSection commentSection . ."
+  "footer footer footer footer footer";
+}
+#header{
+  grid-area: header;
+  align-items: center;
+  text-align: center;
+  border: 1px black;
+  border-color: #143642;
+  color: #143642;
+  margin-bottom: 20px;
+}
+
+#restpage-main{
+  grid-area: main;
+  text-align: justify;
+  border: 2px black;
+  margin-top: 50px;
+  height: 95%;
+  overflow: hidden;
+  border-radius: 20px;
+  grid-column-start: 2;
+  grid-column-end: 4;
+}
+
+#commentSection{
+  grid-area: commentSection;
+  text-indent: 2.5%;
+  text-align: left;
+  line-height: 28px;
+  margin-top: 65px;
+  font-family: 'Monda', sans-serif;
+  border-radius: 20px;
+  background:#DAD2D8;
+  color: #143642;
+  margin-bottom: 20px;
+  grid-column-start: 2;
+  grid-column-end: 4;
+}
+
+#restpage-map{
+  height: 100%;
+  z-index: 0;
+  margin-top: 50px;
+  margin-bottom: 20px;
+  border-radius: 20px;
+  overflow: hidden;
+  grid-column-start: 2;
+  grid-column-end: 4;
+}
+#restpage-navigation{
+  grid-area: navigation;
+  font-family: 'Monda', sans-serif;
+  border-radius: 20px;
+  margin-top: 48px;
+  margin-right: 150px; 
+  height: 160px;
+  z-index: 100;
+  grid-row-start: 1;
+  grid-row-end: 2;
+  grid-column-end: 5;
+}
+
+#drop-content{
+  margin-top: 50px;
+  margin-bottom: 20px;
+}
+
+/* Sidebar List CSS--------------------------------------- */
+#restpage-sidebarList{
+  grid-area: sidebarList;
+  text-align: center;
+  margin-top: 100px;
+  font-family: 'Monda', sans-serif;
+  border-radius: 20px;
+  background:#EC9A29;
+  color: #143642;
+  width: 100%;
+  height: 85%;
+  overflow: hidden;
+}
+#restpage-restName{ 
+  font-size: 18px;
+  font-weight: bolder;
+}
+#restpage-notes {  
+  text-align: left; 
+  margin-left: 10px; 
+  font-size: 16px; 
+  font-weight: bolder;
+} 
+#restpage-phone {  
+  text-align: left; 
+  margin-left: 10px; 
+  font-size: 16px;  
+  font-weight: bolder;
+} 
+#restpage-address{ 
+  text-align: left; 
+  margin-left: 10px; 
+  font-size: 16px;  
+  font-weight: bolder;} 
+
+#restpage-hours{
+  text-align: left; 
+  margin-left: 10px; 
+  font-size: 16px;  
+  font-weight: bolder;
+}
+#restpage-website{
+  text-align: left; 
+  margin-left: 10px; 
+  font-size: 16px;  
+  font-weight: bolder;
+}
```

Comment:
If you have a bunch of rules like this that are just repeats, you can pull this rule out into a class and just write it once in your css and then just add the class to the elements in your html. For example:
```js
.restaurant-font {
  text-align: left;
  margin-left: 10px;
  font-size: 16px;
  font-weight: bolder;
}
```